src/gui/rss/feedlistwidget.cpp

```diff
@@ -235,7 +235,7 @@ QTreeWidgetItem *FeedListWidget::createItem(RSS::Item *rssItem, QTreeWidgetItem
     QIcon icon;
     if (auto feed = qobject_cast<RSS::Feed *>(rssItem)) {
         if (feed->isLoading())
-            icon = QIcon(QStringLiteral(":/icons/loading.png"));
+            icon = UIThemeManager::instance()->getIcon(QStringLiteral(":/icons/loading.png"));
```

Comment:
@jagannatharjun:  Is this line supposed to retain the `:/icons/` path and file extension?  The equivalent change at Line 88 (of this same file) does not.

Reply:
thanks fixed